plugin.yaml

```diff
@@ -10,58 +10,215 @@ plugins:
     # URL to archive containing the plugin or name of directory containing
     # the plugin if it is included in the the blueprint directory under the
     # "plugins" directory.
-    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/master.zip
+    source: https://github.com/szpotona/cloudify-gcp-plugin/archive/add_keypair.zip
 
 node_types:
   cloudify.gcp.nodes.Instance:
     derived_from: cloudify.nodes.Compute
+    properties:
+      use_external_resource:
```

Comment:
It's not implemented. I had to add use external resource because boostrap manager process has some validation and it wasn't passing before. But it's not possible in some points: e.g. in KeyPair there is no sense to put it because Key doesn't have its name.